src/data/nat/log.lean

```diff
@@ -24,6 +25,34 @@ such that `b^k ≤ n`, so if `b^k = n`, it returns exactly `k`. -/
     log (n / b) + 1
   else 0
 
+lemma log_eq_zero {b n : ℕ} (hnb : n < b ∨ b ≤ 1) : log b n = 0 :=
+begin
+  rw [or_iff_not_and_not, not_lt, not_le] at hnb,
+  rw [log, ←ite_not, if_pos hnb],
+end
+
+lemma log_eq_zero_of_lt {b n : ℕ} (hn : n < b) : log b n = 0 :=
+log_eq_zero $ or.inl hn
+
+lemma log_eq_zero_of_lt' {b n : ℕ} (hb : b ≤ 1) : log b n = 0 :=
```

Comment:
Should this not be `of_le`?

Reply:
My bad, had b < 2 before and forgot to change the name